pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -705,7 +705,7 @@
     "rpi_rf" = ps: with ps; [ ]; # missing inputs: rpi-rf
     "rss_feed_template" = ps: with ps; [ aiohttp-cors ];
     "rtorrent" = ps: with ps; [ ];
-    "ruckus_unleashed" = ps: with ps; [ ]; # missing inputs: pyruckus
+    "ruckus_unleashed" = ps: with ps; [ pyruckus ];
```

Comment:
Does this dependency has tests which we can enable?

Reply:
Still interested in why the fail.

Reply:
Because you tried it with 0.14, while home-assistant pins 0.12. These kinds of things matter, because we now have `ruckus_unleashed` support that simply will not work.